examples/yoti_example_flask/docker-compose.yml

```
@@ -2,7 +2,8 @@ version: '3.4'
 services:
   web:
     build:
-      context: ./
+      context: ../../
+      dockerfile: examples/yoti_example_django/Dockerfile
```

```suggestion
      dockerfile: examples/yoti_example_flask/Dockerfile
```

The flask docker-compose file is pointing at the django Dockerfile

---

I presume we don't have a way of differentiating when running the docker images when testing locally do we? Since they should have exactly the same front end

---

The only way I noticed was when I ran the Flask demo, I saw database migrations happen when the docker image was being built which is part of the Django framework.  Other than that, no.  Might be worth changing at least the HTML title page for each framework?

---

Was just thinking Emma would have spotted this when running locally if we had something like that

---

@emmas-yoti not sure if my changeset looks wrong, but looks like this is still pointing to the django dockerfile. Did you test it?